### DISCUSSION TITLE
Unable to Install Plugin for Portable OBS

### DISCUSSION THREAD

**Discussion by JumpinBeans** [sorayuki/obs-multi-rtmp#301] [Q&A]
I have been trying to install the plugin into a portable version of OBS but cannot get OBS to load it.
Using the Zip file, I have extracted it into my OBS folder (ZIP pathing same as OBS pathing). I have also tried creating an install folder in the main OBS folder which I had to do for streamfx.
The installer does not work because it throws an error about "not changing location".

Any help would be appreciated.

[2023-06-22 09-57-47.txt](https://github.com/sorayuki/obs-multi-rtmp/files/11827393/2023-06-22.09-57-47.txt)

**Comment by sorayuki** [comment]
the latest version of this plugin dropped the support of OBS 29.0. try OBS 29.1
________________________________
From: Troy McIntyre ***@***.***>
Sent: Thursday, June 22, 2023 8:44:37 AM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] Unable to Install Plugin for Portable OBS (Discussion #301)


I have been trying to install the plugin into a portable version of OBS but cannot get OBS to load it.
Using the Zip file, I have extracted it into my OBS folder (ZIP pathing same as OBS pathing). I have also tried creating an install folder in the main OBS folder which I had to do for streamfx.
The installer does not work because it throws an error about "not changing location".

Any help would be appreciated.

2023-06-22 09-57-47.txt<https://github.com/sorayuki/obs-multi-rtmp/files/11827393/2023-06-22.09-57-47.txt>

―

**Comment by JumpinBeans** [comment]
Hahaha RIP me, sorta locked into 29.0. for the time being due to another plugin I use being incompatible with 29.1. Hopefully they update it soon and then I will try re-installing this plugin again :)

Thank you for your reply <3

**Comment by sorayuki** [comment]
Would you like to have a try on previous release? Which build for 29.0
________________________________

**Comment by JumpinBeans** [comment]
Happy to try anything, I have my OBS backed up. Currently running 29.0.2, which version would you recommend trying?

**Reply by sorayuki** [reply]
https://github.com/sorayuki/obs-multi-rtmp/releases/tag/0.3.0.2

**Reply by JumpinBeans** [reply]
thank you again <3 will give it a try :)

**Reply by JumpinBeans** [reply]
looks like its all working <3
